sklearn/metrics/tests/test_common.py

```diff
@@ -425,7 +425,8 @@ def precision_recall_curve_padded_thresholds(*args, **kwargs):
     # P = R = F = accuracy in multiclass case
     "micro_f0.5_score", "micro_f1_score", "micro_f2_score",
     "micro_precision_score", "micro_recall_score",
-
+    # log loss is near symmetric with binary predictions
+    "log_loss",
```

Comment:
The reason why this was passing the asymetry test before was due to the high epsilon. As stated in the comment, log loss is near symmetric with binary predictions.